packages/webapp/src/inductions/components/induction-journeys/inviter-witness-journey.tsx

```diff
@@ -0,0 +1,193 @@
+                    memberPreview={memberData}
+                >
+                    <Heading size={1} className="mb-2">
+                        Endorsements
+                    </Heading>
+                    <InductionExpiresIn induction={induction} />
+                    <EndorsementsStatus endorsements={endorsements} />
+                    {userEndorsementIsPending ? (
+                        <InductionEndorsementForm
+                            induction={induction}
+                            setIsReviewingVideo={setIsReviewingVideo}
+                        />
+                    ) : (
+                        <>
+                            <Text>Waiting for all witnesses to endorse.</Text>
+                            <RecommendReview
+                                setIsReviewingVideo={setIsReviewingVideo}
+                            />
+                        </>
+                    )}
+                </Container>
+            );
+        case InductionStatus.PendingDonation:
+            return (
+                <Container
+                    step={
+                        isCommunityActive
+                            ? InductionStepInviter.PendingDonation
+                            : InductionStepGenesis.Donate
+                    }
+                    memberPreview={memberData}
+                >
+                    <Heading size={1} className="mb-2">
+                        Pending donation
+                    </Heading>
+                    <InductionExpiresIn induction={induction} />
+                    <EndorsementsStatus endorsements={endorsements} />
+                    {isCommunityActive ? (
+                        <>
+                            <Text>
+                                This induction is fully endorsed! As soon as the
+                                prospective member completes their donation to
+                                the Eden community, their membership will be
+                                activated and their Eden NFTs will be minted and
+                                distributed.
+                            </Text>
+                            <RecommendReview
+                                setIsReviewingVideo={setIsReviewingVideo}
+                            />
+                        </>
+                    ) : (
+                        <Text>
+                            As soon as this prospective member completes their
+                            donation to the Eden community, their membership is
+                            ready for activation. Once all Genesis members are
+                            fully inducted, memberships will be activated and
+                            Eden NFTs will be distributed.
+                        </Text>
+                    )}
+                </Container>
+            );
+        default:
+            return <p>Unknown error</p>;
+    }
+};
+
+export default InviterWitnessJourney;
```

Comment:
I think like I recommended above for the invitee journey this could benefit from extracting the branching components and logic to its own components.